src/_pytest/stepwise.py

```diff
@@ -0,0 +1,91 @@
+from _pytest.cacheprovider import Cache
+import pytest
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup('general')
+    group.addoption('--sw', '--stepwise', action='store_true', dest='stepwise',
```

Comment:
I think this will cause an option conflict if `pytest-stepwise` is installed.

We can probably deactivate the plugin and emit a warning recommending for users to uninstall pytest-stepwise and use the builtin functionality instead.

@RonnyPfannschmidt might have another suggestion on how to cope/handle this.

Reply:
actually, my fix for that plugin got merged, but not released so it already doesn't work with current pytest versions (raises a typeerror on launch even if the flags aren't set)

Reply:
Yeah, just realized that:

```
  File "c:\pytest\.env36-sw\lib\site-packages\pytest_stepwise\plugin.py", line 16, in pytest_configure
    config.cache = Cache(config)
TypeError: __init__() missing 1 required positional argument: 'config'
```

So I think we can leave as is if pytest-stepwise doesn't plan any further releases.